client/blocks/comment-detail/style.scss

```diff
@@ -12,9 +12,16 @@
 		}
 	}
 
-	&.is-collapsed:hover {
-		box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
-		z-index: 1;
+	&.is-collapsed {
+		&:hover {
+			box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
+			z-index: 1;
+		}
+
+		.accessible-focus &:focus {
+			box-shadow: 0 0 0 1px $blue-medium, 0 0 0 3px $blue-light;
+			z-index: 1;
```

Comment:
🙈 This is probably going to be used all over the place, but let's add this to _z-index.scss, under the `root` context for now.